### DISCUSSION TITLE
Does nugetizer allow to generate NuGet packages without target framework dependencies?

### DISCUSSION THREAD

**Discussion by paulomorgado** [devlooped/nugetizer#452] [Q&A] [answered]
When using [Microsoft.Build.NoTargets](https://github.com/microsoft/MSBuildSdks/tree/main/src/NoTargets) a target framework is always required.

However, that is not the case for all NuGet packages. Some have only content, or MSBuild artifacts or depend on packages that do not have dependencies or do not depend on a specific target framework.

**Comment by kzu** [comment]
If you have no package references, no dependencies will be generated. You can also set `Pack=false` on references and they won't be packed at all.

**Reply by paulomorgado** [reply]
What if I have package references that depend on no projects but depend on other packages that have no target framework? Is a target framework still required?

**Reply by kzu** [reply]
There is no such thing as a project without a TF. That's an MSBuild/SDK limitation, AFAIK.

**Reply by paulomorgado** [reply, marked as answer]
> There is no such thing as a project without a TF. That's an MSBuild/SDK limitation, AFAIK.

Yeap!
